cdap-app-fabric/src/main/java/org/apache/twill/yarn/YarnTwillController.java

```diff
@@ -311,4 +323,14 @@ public ResourceReport getResourceReport() {
     // in case the user calls this before starting, return null
     return (resourcesClient == null) ? null : resourcesClient.get();
   }
+
+  // begin change CDAP-5135
```

Comment:
these comments are just to help see what is different from Apache Twill. There is already a TODO at the top to remove the entire class once the changes are made in Twill.